.github/autobuild/mac.sh

```diff
@@ -30,20 +30,35 @@ prepare_signing() {
     [[ -n "${MACOS_CERTIFICATE:-}" ]] || return 1
     [[ -n "${MACOS_CERTIFICATE_ID:-}" ]] || return 1
     [[ -n "${MACOS_CERTIFICATE_PWD:-}" ]] || return 1
+    [[ -n "${MACAPP_CERTIFICATE:-}" ]] || return 1
+    [[ -n "${MACAPP_CERTIFICATE_ID:-}" ]] || return 1
+    [[ -n "${MACAPP_CERTIFICATE_PWD:-}" ]] || return 1
     [[ -n "${NOTARIZATION_PASSWORD:-}" ]] || return 1
     [[ -n "${KEYCHAIN_PASSWORD:-}" ]] || return 1
 
     echo "Signing was requested and all dependencies are satisfied"
 
-    # Put the cert to a file
-    echo "${MACOS_CERTIFICATE}" | base64 --decode > certificate.p12
+    ## Put the certs to a file
+    # MACOS_CERTIFICATE - Mac Installer Distribution
+    # MACAPP_CERTIFICATE - Mac App Distribution
+    echo "${MACOS_CERTIFICATE}" | base64 --decode > macinst_certificate.p12
```

Comment:
It looks like you're repurposing the existing signing cert for the App Store - this will break the non-app store signing. 
Can you separate out the new requisite certs and name the env vars appropriately?